pcsx2/x86/microVU_Lower.inl

```diff
@@ -1101,16 +1174,31 @@ mVUop(mVU_ILW)
 	{
 		void* ptr = mVU.regs().Mem + offsetSS;
 		std::optional<xAddressVoid> optaddr(mVUoptimizeConstantAddr(mVU, _Is_, _Imm11_, offsetSS));
-		if (!optaddr.has_value())
+		if (EmuConfig.Gamefixes.IbitHack || !optaddr.has_value())
 		{
 			mVU.regAlloc->moveVIToGPR(gprT1, _Is_);
-			if (_Imm11_ != 0)
-				xADD(gprT1, _Imm11_);
+			if (!EmuConfig.Gamefixes.IbitHack)
+			{
+				if (_Imm11_ != 0)
+					xADD(gprT1, _Imm11_);
+			}
+			else
+			{
+				xMOV(gprT2, ptr32[&curI]);
```

Comment:
If I'm reading the encoding correctly (i.e. `sign_Extend_11bit(code)`, this could also just be
```asm
mov gprT2, dword [&curI]
shl gprT2, 21
sar gprT2, 21
```

and ditto for the others.